### PR TITLE
feat: support any <pre> tag which has @startuml and @enduml

### LINKED ISSUE #546: feat: support any <pre> tag which has @startuml and @enduml
To support various web pages, make this plugin transform any `<pre>` tag which has a text with the prefix `@startuml` and the suffix `@enduml` on any url.
(I'm not sure we can target any url ... ?)

e.g. https://github.com/Melandel/sandbox-for-markdown-preview-with-plantuml-support/blob/master/generated-with-default-theme.html#L689-L702 (from #518)

### PR DESCRIPTION
**Pull request by emtsubasa**
close #546
alleviate (?) #518

### CODE REVIEW

**Comment by kumachan-mis** [comment]
close #508 too

File: manifest.json [hunk 1]
```diff
@@ -18,17 +18,7 @@
...
-        "*://*/*.pu*",
-        "*://*/*.puml*",
-        "*://*/*.plantuml*",
-        "*://*/*.wsd*",
-        "file:///*/*.pu*",
-        "file:///*/*.puml*",
-        "file:///*/*.plantuml*",
-        "file:///*/*.wsd*"
-      ],
+      "matches": ["https://*/*", "http://*/*", "file:///*/*", "*://*/*"],
```
Comment (kumachan-mis) [inline manifest.json:21]: `"*://*/*"` should be removed (We now consider only http, https and file scheme)

File: src/finder/codeBlockFinder.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@ import { Constants } from '../constants';
 import { CodeFinder, UmlCodeContent } from './finder';
 import { extractSubIncludedText } from './finderUtil';
 
-export class RawFileFinder implements CodeFinder {
-  private readonly URL_REGEX = /^.*\.(plantuml|pu|puml|wsd)(\?.*)?$/;
+export class CodeBlockFinder implements CodeFinder {
+  private readonly URL_REGEX = /(file)|(https?)|([A-Z]):\/\/.+/;
```
Comment (kumachan-mis) [inline src/finder/codeBlockFinder.ts:7]: windows file URI is like `file://C:/.+` NOT `C://.+` ... (really confusing!)
so `/(file)|(https?):\/\/.+/` is fine

Comment (exKAZUu) [reply src/finder/codeBlockFinder.ts:7]: How about `/(file|https?):\/\/.+/` ?

Comment (kumachan-mis) [reply src/finder/codeBlockFinder.ts:7]: better one!

Comment (exKAZUu) [reply src/finder/codeBlockFinder.ts:7]: `/(file)|(https?):\/\/.+/.test("file")` yields `true`, so I think this must be `/(file|https?):\/\/.+/`

Comment (exKAZUu) [reply src/finder/codeBlockFinder.ts:7]: c.f. https://www.boost.org/doc/libs/1_56_0/libs/regex/doc/html/boost_regex/syntax/basic_extended.html#boost_regex.syntax.basic_extended.operator_precedence

**Pull request merged**